exporter/kafkaexporter/README.md

```diff
@@ -26,6 +26,7 @@ The following settings are required:
 The following settings can be optionally configured:
 - `brokers` (default = localhost:9092): The list of kafka brokers.
 - `resolve_canonical_bootstrap_servers_only` (default = false): Whether to resolve then reverse-lookup broker IPs during startup.
+- `client_id` (default = "sarama"): The client ID to configure the Sarama Kafka client with. The client ID will be used for all produce requests.
```

Comment:
If we set default to `sarama`, I thinks this may be a breaking change for users? 
For example, when user upgrade the collector in new version but without changing the config to specify `client_id`, then all the request will be added with a client_id `sarama`, which maybe result in user's requests being throttled.
So I think maybe we should keep original logic, if `client_id` is not specified, the collector do not add `client_id` in saram kafka client.
Or maybe we should change this pr's change_type from enhancement to breaking.

Reply:
@fatsheep9146 That's why I left it out of the P.R originally, but it's actually not a breaking change because `sarama` is already being used since its the default in the sarama library: https://github.com/IBM/sarama/blob/main/config.go#L16

So this will be a no-op change for people who don't manually configure it.

Reply:
> @fatsheep9146 That's why I left it out of the P.R originally, but it's actually not a breaking change because `sarama` is already being used since its the default in the sarama library: https://github.com/IBM/sarama/blob/main/config.go#L16
> 
> So this will be a no-op change for people who don't manually configure it.

Ok! That make senses to me.